cypress/support/commands.js

```diff
@@ -36,7 +36,7 @@ Cypress.Commands.add('install', function() {
 	cy.get('input[id=additionalLocales-fr_CA').check();
 
 	// Complete the installation
-	cy.get('button[id^=submitFormButton-]').click();
+	cy.get('button[id^=submitFormButton-]', {timeout: 90000}).click();
```

Comment:
@withanage, this won't extend the timeout on the `click()` call, it only extends the timeout on the `cy.get` call. The default timeout should be long enough for the tests to pass (and Travis gets through it OK). Are tests breaking somewhere on Travis, or on your local setup?

Reply:
@asmecher  sorry, I  have overiddden it and corrected it again. this can be ignored.